app/celery/scheduled_tasks.py

```diff
@@ -186,25 +186,39 @@ def replay_created_notifications():
 @statsd(namespace="tasks")
 def check_precompiled_letter_state():
     letters = dao_precompiled_letters_still_pending_virus_check()
-    letter_ids = [str(letter.id) for letter in letters]
 
     if len(letters) > 0:
-        current_app.logger.exception(
-            '{} precompiled letters have been pending-virus-check for over 90 minutes. Notifications: {}'.format(
-                len(letters),
-                letter_ids)
-        )
+        letter_ids = [str(letter.id) for letter in letters]
+
+        msg = "{} precompiled letters have been pending-virus-check for over 90 minutes. " \
+              "Notifications: {}".format(len(letters), letter_ids)
+
+        current_app.logger.exception(msg)
+
+        if current_app.config['NOTIFY_ENVIRONMENT'] in ['live', 'production', 'test']:
```

Comment:
Test is here so that we can test the creation of the Zendesk ticket (its message, subject and ticket type) in our tests. We don't really send any Zendesk tickets though in the test environment because we mock out the Zendesk client.